pages/reset-password.js

```diff
@@ -0,0 +1,36 @@
+import Image from 'next/image';
+
+import Background from '@/components/Background';
+import BackButton from '@/components/BackButton';
+import PasswordResetForm from '@/components/forms/PasswordReset';
+
+import logo from '@/public/logo-large.svg';
+
+export const passwordResetPageTestIds = {
+  headlineText: 'password-reset__headline-text',
+};
+
+const PasswordReset = () => {
+  return (
+    <Background imagePath="/background.png">
+      <div className="h-full">
+        <div className="pl-8 pt-8">
+          <BackButton />
```

Comment:
Just notice that this button width expands to fit the whole screen so should we set the width and height?

The current style | The design
--- | ---
![Screen Shot 2565-04-01 at 10 35 43](https://user-images.githubusercontent.com/1772999/161190509-6e1a57fb-4b5f-4edc-9f11-883ec319cb0c.png) | ![Screen Shot 2565-04-01 at 10 36 58](https://user-images.githubusercontent.com/1772999/161190503-fc4e7944-c91a-4a61-8951-556115ebecf2.png)

Reply:
done in dbf9d1e261944188cc87b0d6f495a313bea9b890